File: .ci/doc/templates/blank.tpl.js
// load the Kuzzle SDK module
const
  {
    Kuzzle,
    WebSocket,
    Http
  } = require('kuzzle-sdk');

const kuzzle = new Kuzzle(
  new WebSocket('kuzzle', { pingInterval: 2000 })
);

[snippet-code]
console.log('Everything is ok');

File: doc-framework/test/templates/blank.tpl.go
package main

import (
	"fmt"
	"encoding/json"

	"github.com/kuzzleio/sdk-go/protocol/websocket"
	"github.com/kuzzleio/sdk-go/kuzzle"
	"github.com/kuzzleio/sdk-go/types"
)

func main() {
	[snippet-code]
	k.Server.Now(nil)
	fmt.Println("Everything is ok")
}
